src/main/java/demo/userboard/controller/UserController.java

```diff
@@ -21,4 +25,13 @@ public ResponseEntity<JoinResponseDto> joinUser(@RequestBody JoinRequestDto join
         return ResponseEntity.ok(userService.join(joinRequestDto));
     }
 
+    @PostMapping("/login")
+    public ResponseEntity<LoginResponseDto> loginUser(@RequestBody LoginRequestDto loginRequestDto) {
+        LoginResponseDto responseDto = userService.login(loginRequestDto);
+        if (responseDto == null) {
+            throw new CustomException(CustomErrorCode.UNAUTHORIZED);
+        } else {
+            return ResponseEntity.ok(responseDto);
+        }
+    }
```

Comment:
지난번에도 비슷한 이야기 드렸던것 같습니다
로그인을 하는 주체는 userService에 login이기 때문에
로그인이 성공했다 실패했다는 컨트롤러에서 확인할 필요가 없습니다.

때문에 해당 로직은 컨트롤러가 아니라 Service 레이어로 들어가는게 맞습니다
그리고 이전 pr에서 공통 응답값 적용했던 부분까지 같이 적용한다면 아래와 같아지겠네요 

```suggestion
    @PostMapping("/login")
    public CommonResponse<LoginResponseDto> loginUser(@RequestBody LoginRequestDto loginRequestDto) {

        return success(userService.login(loginRequestDto));
    }
```

Reply:
@evan-song2 컨트롤부분 수정 완료했습니다. [ec89b73](https://github.com/PNoahKR/user-board/pull/8/commits/ec89b734f642559e7299622912483113ceebe933)